bin/node/src/chain_spec/argo_bridge_config.rs

```diff
@@ -0,0 +1,21 @@
+use node_runtime::{argo_bridge::types::BridgeStatus, ArgoBridgeConfig};
+
+pub fn production_config() -> ArgoBridgeConfig {
```

Comment:
I also think that you can set the mint allowance storage variable to be `0` by default since I don't think any other initial value is plausible